mona_openai/mona_openai.py

```diff
@@ -1,13 +1,11 @@
 import time
-import logging
+from .loggers.mona_logger.mona_logger import MonaLogger
```

Comment:
BTW, regarding all imports - I personally think it's better to use absolute paths (and not relative) to ensure the correct file is always imported and to avoid possible import errors

Reply:
I'm actually fond of these relative paths. It allows differentiating between global imported packages and local modules. Can you explain what is your concern here?